torch/cuda/streams.py

```diff
@@ -203,4 +203,8 @@ def __repr__(self):
         else:
             return '<torch.cuda.Event uninitialized>'
 
-_Graph = torch._C._CudaGraphBase
+# We could give methods default arguments in the Pybind11 (torch/csrc/cuda/Graph.cpp)
+# but it's more readable to set default arguments here.
```

Comment:
More... readable? I thought pybind11 makes setting defaults pretty easy. Also, if you do it this way, you pay for all the Python opcodes to run the Python `capture_begin`.

Reply:
Pybind11 lets you set defaults easily, but to the average user browsing through pytorch, Python is much more accessible than C++. I guess looking through the code to see what it does becomes irrelevant if and when this call is publicly documented.

Reply:
hopefully done